adobecom/milo
# MWPW-130326 - Footer Fragments

* Can spec a meta tag to load a footer
* Locale aware
* Metadata.xlsx can be used for inheritance
* Metadta block can be used for one offs

Resolves: [MWPW-130326](https://jira.corp.adobe.com/browse/MWPW-130326)

---

**How to test**
1. Create a promo fragment in `/fragments/footer-promos/{{name-of-promo}}`.
2. Use `footer-promo-tag` in metadata (xlsx or block) with `{{name-of-promo}}`.
3. Observe: Promo content will load just above footer in the last section of a page.
4. Can also be done in-region... /de/, /fr/, etc.
5. Note: this is expected to live outside any authored carousels or tabs.

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/cmillar/footer-fragment-promo-thing?martech=off
- After: https://footer-fragment--milo--adobecom.hlx.page/drafts/cmillar/footer-fragment-promo-thing?martech=off
